doc/source/dev/conduct/code_of_conduct.rst

```diff
@@ -133,8 +133,13 @@ of Conduct Committee, and the SciPy Steering Committee (if involved), are
 obligated to protect the identity of the reporter, and to treat the content of
 complaints with confidentiality (unless discussed with the reporter).
 
-The process for acting on any received code of conduct violation report at a
-high level (for all but the most severe cases) will be:
+There are two tracks of response; the first is for severe and obvious
+breaches, where the communication involves personal threat or violent, sexist
+or racist language.  We will immediately disconnect the originator from Scipy
+communication channels in this case; please see the manual for details.
+
+In cases that are less extreme, the process for acting on any received code of
```

Comment:
This still qualifies the cases that @njsmith thinks are the most common ones as "extreme". I think the split made here is good but will try to find more neutral language.

Reply:
It also needs to make clear that which avenue to choose is up to the reporter - it's not the case that anything less than sexist/racist language requires mediation.

Reply:
Of all of these:

    - Violent threats or language directed against another person.
    - Sexist, racist, or otherwise discriminatory jokes and language.
    - Posting sexually explicit or violent material.
    - Posting (or threatening to post) other people's personally identifying information ("doxing").
    - Sharing private content, such as emails sent privately or non-publicly, or unlogged forums such as IRC channel history, without the sender's consent.
    - Personal insults, especially those using racist or sexist terms.
    - Unwelcome sexual attention.
    - Excessive profanity. Please avoid swearwords; people differ greatly in their sensitivity to swearing.
    - Repeated harassment of others. In general, if someone asks you to stop, then stop.
    - Advocating for, or encouraging, any of the above behaviour.

it's probably only excessive profanity, sharing private content and perhaps repeated harassment that we'd want to see follow the discussion-mediation route.